test/runtests.jl

```diff
@@ -35,6 +35,19 @@ cd("..")
 
 cd("modelling")
 @testset "Modelling Examples" begin
+    include("modelling/n-queens.jl")
+    @test solution == [
```

Comment:
is there a unique solution ?
If not be aware that the solver could return another one and make the test fail, it may be safer to test the objective value and not solution

Reply:
Great point. There is more than one feasible solution and different solvers find different solutions. I'll fix this : )

Reply:
great, let me know when you make the change

Reply:
Okay, I fixed this. The tests will now check that any feasible solution is found rather than the one solution that GLPK produces.